unlock-js/src/unlockProvider.js

```diff
@@ -57,6 +58,18 @@ export default class UnlockProvider extends providers.JsonRpcProvider {
     return this[method](params)
   }
 
+  /**
+   * Implementation of personal_sign JSON-RPC call
+   * @param {string} data the data to sign.
+   * @param {string} _ the address to sign it with -- ignored because
+   * we use the address in this class.
+   */
+  // eslint-disable-next-line no-unused-vars
+  personal_sign([data, _]) {
```

Comment:
This might be from another PR but we need to use camelCase ;)

Reply:
Camel case would be incompatible with the json-rpc method, since this is an implementation of https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_sign